compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -250,6 +294,71 @@ struct InsertQuantizeOp final : public luci::CircleNodeMutableVisitor<void>
   INSERT_QUANTIZE_TO_BINARY_OP(luci::CirclePow, x, y)
   INSERT_QUANTIZE_TO_BINARY_OP(luci::CircleSub, x, y)
 
+  // Multiple-output ops that receive one activation as inputs
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleSplit, input)
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleSplitV, input)
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleTopKV2, input)
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleUnique, input)
+  INSERT_QUANTIZE_TO_UNARY_MULTI_OUTPUT_OP(luci::CircleUnpack, value)
+
+  // Multiple-output ops that receive multiple activations as inputs
+  INSERT_QUANTIZE_MULTI_OUTPUT_NODES_WITH_INPUTS(luci::CircleIf, input, input_count)
+  INSERT_QUANTIZE_MULTI_OUTPUT_NODES_WITH_INPUTS(luci::CircleWhile, input, input_count)
+  INSERT_QUANTIZE_MULTI_OUTPUT_NODES_WITH_INPUTS(luci::CircleCustom, inputs, numInputs)
```

Comment:
Hmm.. I think it is very unlikely to quantize If/While/Custom Ops (+ I'm not sure this is a right quantization scheme). Could you remove these Ops? We may support these Ops when necessary.